Documents/Command/AWS-RunAnsible/AWS-RunAnsible.yaml

```diff
@@ -0,0 +1,189 @@
+schemaVersion: '2.2'
+description: 'Use this document to run Ansible playbooks on Amazon EC2 managed instances.
+  Specify either playbook YAML text, playbook URL, or URL. If you specify multiple
+  values, precedence order is: url, PlaybookURL, and then playbook. If you specify
+  URL, everything from that point and below in S3 will be synced to the local machine
+  (including any roles and custom modules). Use PlaybookFile for the playbook to run
+  from that URL. Use the extravar parameter to send runtime variables to the Ansible
+  execution. Use the check parameter to perform a dry run of the Ansible execution.
+  The output of the dry run shows the changes that will be made when the playbook
+  is executed. This version also supports zip files via http or s3 URLs. It also supports
+  providing an s3 URL for a directory containing the playbook files. '
+parameters:
+  playbook:
+    type: String
+    description: (Optional) If you don't specify a URL, then you must specify playbook
+      YAML in this field.
+    default: ''
+    displayType: textarea
+  PlaybookURL:
```

Comment:
The current published version (https://console.aws.amazon.com/systems-manager/documents/AWS-RunAnsiblePlaybook/content?region=us-east-1) uses parameter name "playbookurl". Let's keep the same for backward compatibility.